### PR TITLE
Remove travis directory

### PR DESCRIPTION
**Pull request by liyuqian**
This reflects that we no longer uses travis. Scripts are moved to ci folder.

### CODE REVIEW

File: tools/licenses/lib/main.dart [hunk 1]
```diff
@@ -54,7 +54,7 @@ abstract class RepositoryLicensedFile extends RepositoryFile {
   static final RegExp _readmeNamePattern = new RegExp(r'\b_*(?:readme|contributing|patents)_*\b', caseSensitive: false);
   static final RegExp _buildTimePattern = new RegExp(r'^(?!.*gen$)(?:CMakeLists\.txt|(?:pkgdata)?Makefile(?:\.inc)?(?:\.am|\.in|)|configure(?:\.ac|\.in)?|config\.(?:sub|guess)|.+\.m4|install-sh|.+\.sh|.+\.bat|.+\.pyc?|.+\.pl|icu-configure|.+\.gypi?|.*\.gni?|.+\.mk|.+\.cmake|.+\.gradle|.+\.yaml|pubspec\.lock|\.packages|vms_make\.com|pom\.xml|\.project|source\.properties)$', caseSensitive: false);
   static final RegExp _docsPattern = new RegExp(r'^(?:INSTALL|NEWS|OWNERS|AUTHORS|ChangeLog(?:\.rst|\.[0-9]+)?|.+\.txt|.+\.md|.+\.log|.+\.css|.+\.1|doxygen\.config|.+\.spec(?:\.in)?)$', caseSensitive: false);
-  static final RegExp _devPattern = new RegExp(r'^(?:codereview\.settings|.+\.~|.+\.~[0-9]+~|\.clang-format|\.gitattributes|\.landmines|\.DS_Store|\.travis\.yml)$', caseSensitive: false);
+  static final RegExp _devPattern = new RegExp(r'^(?:codereview\.settings|.+\.~|.+\.~[0-9]+~|\.clang-format|\.gitattributes|\.landmines|\.DS_Store|\.travis\.yml|\.cirrus\.yml)$', caseSensitive: false);
```
Comment (gspencergoog) [inline tools/licenses/lib/main.dart]: You could remove travis.yml from this now.

**Comment by gspencergoog** [comment]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: tools/licenses/lib/main.dart (hunk 1 above)
Comment (liyuqian) [reply tools/licenses/lib/main.dart]: Not yet. Although we don't have travis.yml, plenty of third party libs still have travis.yml and we don't want them to be in our licenses golden.

**Comment by liyuqian** [comment]
The commit caused our Linux Chrome Engine bots to fail. https://chromium-review.googlesource.com/c/chromium/tools/build/+/1164636 should fix it.

**Pull request merged**